modules/core/src/test/java/org/apache/ignite/spi/communication/tcp/IgniteTcpCommunicationHandshakeWaitTest.java

```diff
@@ -69,8 +69,8 @@
         TcpCommunicationSpi commSpi = new TcpCommunicationSpi();
 
         commSpi.setConnectTimeout(COMMUNICATION_TIMEOUT);
-        commSpi.setMaxConnectTimeout(COMMUNICATION_TIMEOUT);
-        commSpi.setReconnectCount(1);
+        commSpi.setMaxConnectTimeout(4 * COMMUNICATION_TIMEOUT);
```

Comment:
Please do not change existing tests which are not related to forcible node killing.